templates/layout/page.html.twig

```diff
@@ -130,9 +138,9 @@
 
     <a id="main-content" tabindex="-1"></a>
 
-    {% if page.messages %}
-      {{ page.messages }}
-    {% endif %}
+{#    {% if page.messages %}#}
```

Comment:
Why is this commented out?

If we really do not need it anymore, lets delete the rows fully. If we do, we should have a comment here explaining when these should be uncommented.

Reply:
Thank you for always being so observant!! 🌹  This was a mistake that I should have never commited. Removed the comments.